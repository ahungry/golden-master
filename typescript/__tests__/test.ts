
import * as SUT_example_math_ts from '../example/math.ts'

describe('example/math.ts', () => {
  
  describe('add', () => {
    it('Will match a known snapshot thanks to golden master (4, -9).', async () => {
      const result = await SUT_example_math_ts.add(4, -9)
      expect(result).toMatchSnapshot()
    })
  })

  describe('add2', () => {
    it('Will match a known snapshot thanks to golden master (10, -5).', async () => {
      const result = await SUT_example_math_ts.add2(10, -5)
      expect(result).toMatchSnapshot()
    })
  })

  describe('add3', () => {
    it('Will match a known snapshot thanks to golden master (-5, 8).', async () => {
      const result = await SUT_example_math_ts.add3(-5, 8)
      expect(result).toMatchSnapshot()
    })
  })

  describe('adder', () => {
    it('Will match a known snapshot thanks to golden master (8, 4).', async () => {
      const result = await SUT_example_math_ts.adder(8, 4)
      expect(result).toMatchSnapshot()
    })
  })

  describe('MyMath.blub', () => {
    it('Will match a known snapshot thanks to golden master ("dog").', async () => {
      const result = await SUT_example_math_ts.MyMath.blub('dog')
      expect(result).toMatchSnapshot()
    })
  })

  describe('MyMath.add', () => {
    it('Will match a known snapshot thanks to golden master ().', async () => {
      const result = await SUT_example_math_ts.MyMath.add()
      expect(result).toMatchSnapshot()
    })
  })

})

import * as SUT_example_words_ts from '../example/words.ts'

describe('example/words.ts', () => {
  
  describe('add', () => {
    it('Will match a known snapshot thanks to golden master ("dog", "dog").', async () => {
      const result = await SUT_example_words_ts.add('dog', 'dog')
      expect(result).toMatchSnapshot()
    })
  })

  describe('add2', () => {
    it('Will match a known snapshot thanks to golden master ("dog", "dog").', async () => {
      const result = await SUT_example_words_ts.add2('dog', 'dog')
      expect(result).toMatchSnapshot()
    })
  })

  describe('add3', () => {
    it('Will match a known snapshot thanks to golden master ("dog", "dog").', async () => {
      const result = await SUT_example_words_ts.add3('dog', 'dog')
      expect(result).toMatchSnapshot()
    })
  })

  describe('adder', () => {
    it('Will match a known snapshot thanks to golden master ("dog", "dog").', async () => {
      const result = await SUT_example_words_ts.adder('dog', 'dog')
      expect(result).toMatchSnapshot()
    })
  })

  describe('MyWords.blub', () => {
    it('Will match a known snapshot thanks to golden master ("dog").', async () => {
      const result = await SUT_example_words_ts.MyWords.blub('dog')
      expect(result).toMatchSnapshot()
    })
  })

  describe('MyWords.add', () => {
    it('Will match a known snapshot thanks to golden master ().', async () => {
      const result = await SUT_example_words_ts.MyWords.add()
      expect(result).toMatchSnapshot()
    })
  })

})

