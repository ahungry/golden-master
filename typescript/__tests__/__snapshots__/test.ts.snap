// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`example/math.ts add Will match a known snapshot (-1,-1). 1`] = `-2`;

exports[`example/math.ts add Will match a known snapshot (-1,0). 1`] = `-1`;

exports[`example/math.ts add Will match a known snapshot (-1,1). 1`] = `0`;

exports[`example/math.ts add Will match a known snapshot (-1,42). 1`] = `41`;

exports[`example/math.ts add Will match a known snapshot (-1,42). 2`] = `41`;

exports[`example/math.ts add Will match a known snapshot (-1,42). 3`] = `41`;

exports[`example/math.ts add Will match a known snapshot (-1,NaN). 1`] = `NaN`;

exports[`example/math.ts add Will match a known snapshot (0,-1). 1`] = `-1`;

exports[`example/math.ts add Will match a known snapshot (0,0). 1`] = `0`;

exports[`example/math.ts add Will match a known snapshot (0,1). 1`] = `1`;

exports[`example/math.ts add Will match a known snapshot (0,42). 1`] = `42`;

exports[`example/math.ts add Will match a known snapshot (0,42). 2`] = `42`;

exports[`example/math.ts add Will match a known snapshot (0,42). 3`] = `42`;

exports[`example/math.ts add Will match a known snapshot (0,NaN). 1`] = `NaN`;

exports[`example/math.ts add Will match a known snapshot (1,-1). 1`] = `0`;

exports[`example/math.ts add Will match a known snapshot (1,0). 1`] = `1`;

exports[`example/math.ts add Will match a known snapshot (1,1). 1`] = `2`;

exports[`example/math.ts add Will match a known snapshot (1,42). 1`] = `43`;

exports[`example/math.ts add Will match a known snapshot (1,42). 2`] = `43`;

exports[`example/math.ts add Will match a known snapshot (1,42). 3`] = `43`;

exports[`example/math.ts add Will match a known snapshot (1,NaN). 1`] = `NaN`;

exports[`example/math.ts add Will match a known snapshot (42,-1). 1`] = `41`;

exports[`example/math.ts add Will match a known snapshot (42,-1). 2`] = `41`;

exports[`example/math.ts add Will match a known snapshot (42,-1). 3`] = `41`;

exports[`example/math.ts add Will match a known snapshot (42,0). 1`] = `42`;

exports[`example/math.ts add Will match a known snapshot (42,0). 2`] = `42`;

exports[`example/math.ts add Will match a known snapshot (42,0). 3`] = `42`;

exports[`example/math.ts add Will match a known snapshot (42,1). 1`] = `43`;

exports[`example/math.ts add Will match a known snapshot (42,1). 2`] = `43`;

exports[`example/math.ts add Will match a known snapshot (42,1). 3`] = `43`;

exports[`example/math.ts add Will match a known snapshot (42,42). 1`] = `84`;

exports[`example/math.ts add Will match a known snapshot (42,42). 2`] = `84`;

exports[`example/math.ts add Will match a known snapshot (42,42). 3`] = `84`;

exports[`example/math.ts add Will match a known snapshot (42,42). 4`] = `84`;

exports[`example/math.ts add Will match a known snapshot (42,42). 5`] = `84`;

exports[`example/math.ts add Will match a known snapshot (42,42). 6`] = `84`;

exports[`example/math.ts add Will match a known snapshot (42,42). 7`] = `84`;

exports[`example/math.ts add Will match a known snapshot (42,42). 8`] = `84`;

exports[`example/math.ts add Will match a known snapshot (42,42). 9`] = `84`;

exports[`example/math.ts add Will match a known snapshot (42,NaN). 1`] = `NaN`;

exports[`example/math.ts add Will match a known snapshot (42,NaN). 2`] = `NaN`;

exports[`example/math.ts add Will match a known snapshot (42,NaN). 3`] = `NaN`;

exports[`example/math.ts add Will match a known snapshot (NaN,-1). 1`] = `NaN`;

exports[`example/math.ts add Will match a known snapshot (NaN,0). 1`] = `NaN`;

exports[`example/math.ts add Will match a known snapshot (NaN,1). 1`] = `NaN`;

exports[`example/math.ts add Will match a known snapshot (NaN,42). 1`] = `NaN`;

exports[`example/math.ts add Will match a known snapshot (NaN,42). 2`] = `NaN`;

exports[`example/math.ts add Will match a known snapshot (NaN,42). 3`] = `NaN`;

exports[`example/math.ts add Will match a known snapshot (NaN,NaN). 1`] = `NaN`;

exports[`example/math.ts add2 Will match a known snapshot (-1,-1). 1`] = `-2`;

exports[`example/math.ts add2 Will match a known snapshot (-1,0). 1`] = `-1`;

exports[`example/math.ts add2 Will match a known snapshot (-1,1). 1`] = `0`;

exports[`example/math.ts add2 Will match a known snapshot (-1,42). 1`] = `41`;

exports[`example/math.ts add2 Will match a known snapshot (-1,42). 2`] = `41`;

exports[`example/math.ts add2 Will match a known snapshot (-1,42). 3`] = `41`;

exports[`example/math.ts add2 Will match a known snapshot (-1,NaN). 1`] = `NaN`;

exports[`example/math.ts add2 Will match a known snapshot (0,-1). 1`] = `-1`;

exports[`example/math.ts add2 Will match a known snapshot (0,0). 1`] = `0`;

exports[`example/math.ts add2 Will match a known snapshot (0,1). 1`] = `1`;

exports[`example/math.ts add2 Will match a known snapshot (0,42). 1`] = `42`;

exports[`example/math.ts add2 Will match a known snapshot (0,42). 2`] = `42`;

exports[`example/math.ts add2 Will match a known snapshot (0,42). 3`] = `42`;

exports[`example/math.ts add2 Will match a known snapshot (0,NaN). 1`] = `NaN`;

exports[`example/math.ts add2 Will match a known snapshot (1,-1). 1`] = `0`;

exports[`example/math.ts add2 Will match a known snapshot (1,0). 1`] = `1`;

exports[`example/math.ts add2 Will match a known snapshot (1,1). 1`] = `2`;

exports[`example/math.ts add2 Will match a known snapshot (1,42). 1`] = `43`;

exports[`example/math.ts add2 Will match a known snapshot (1,42). 2`] = `43`;

exports[`example/math.ts add2 Will match a known snapshot (1,42). 3`] = `43`;

exports[`example/math.ts add2 Will match a known snapshot (1,NaN). 1`] = `NaN`;

exports[`example/math.ts add2 Will match a known snapshot (42,-1). 1`] = `41`;

exports[`example/math.ts add2 Will match a known snapshot (42,-1). 2`] = `41`;

exports[`example/math.ts add2 Will match a known snapshot (42,-1). 3`] = `41`;

exports[`example/math.ts add2 Will match a known snapshot (42,0). 1`] = `42`;

exports[`example/math.ts add2 Will match a known snapshot (42,0). 2`] = `42`;

exports[`example/math.ts add2 Will match a known snapshot (42,0). 3`] = `42`;

exports[`example/math.ts add2 Will match a known snapshot (42,1). 1`] = `43`;

exports[`example/math.ts add2 Will match a known snapshot (42,1). 2`] = `43`;

exports[`example/math.ts add2 Will match a known snapshot (42,1). 3`] = `43`;

exports[`example/math.ts add2 Will match a known snapshot (42,42). 1`] = `84`;

exports[`example/math.ts add2 Will match a known snapshot (42,42). 2`] = `84`;

exports[`example/math.ts add2 Will match a known snapshot (42,42). 3`] = `84`;

exports[`example/math.ts add2 Will match a known snapshot (42,42). 4`] = `84`;

exports[`example/math.ts add2 Will match a known snapshot (42,42). 5`] = `84`;

exports[`example/math.ts add2 Will match a known snapshot (42,42). 6`] = `84`;

exports[`example/math.ts add2 Will match a known snapshot (42,42). 7`] = `84`;

exports[`example/math.ts add2 Will match a known snapshot (42,42). 8`] = `84`;

exports[`example/math.ts add2 Will match a known snapshot (42,42). 9`] = `84`;

exports[`example/math.ts add2 Will match a known snapshot (42,NaN). 1`] = `NaN`;

exports[`example/math.ts add2 Will match a known snapshot (42,NaN). 2`] = `NaN`;

exports[`example/math.ts add2 Will match a known snapshot (42,NaN). 3`] = `NaN`;

exports[`example/math.ts add2 Will match a known snapshot (NaN,-1). 1`] = `NaN`;

exports[`example/math.ts add2 Will match a known snapshot (NaN,0). 1`] = `NaN`;

exports[`example/math.ts add2 Will match a known snapshot (NaN,1). 1`] = `NaN`;

exports[`example/math.ts add2 Will match a known snapshot (NaN,42). 1`] = `NaN`;

exports[`example/math.ts add2 Will match a known snapshot (NaN,42). 2`] = `NaN`;

exports[`example/math.ts add2 Will match a known snapshot (NaN,42). 3`] = `NaN`;

exports[`example/math.ts add2 Will match a known snapshot (NaN,NaN). 1`] = `NaN`;

exports[`example/math.ts add3 Will match a known snapshot (-1,-1). 1`] = `-2`;

exports[`example/math.ts add3 Will match a known snapshot (-1,0). 1`] = `-1`;

exports[`example/math.ts add3 Will match a known snapshot (-1,1). 1`] = `0`;

exports[`example/math.ts add3 Will match a known snapshot (-1,42). 1`] = `41`;

exports[`example/math.ts add3 Will match a known snapshot (-1,42). 2`] = `41`;

exports[`example/math.ts add3 Will match a known snapshot (-1,42). 3`] = `41`;

exports[`example/math.ts add3 Will match a known snapshot (-1,NaN). 1`] = `NaN`;

exports[`example/math.ts add3 Will match a known snapshot (0,-1). 1`] = `-1`;

exports[`example/math.ts add3 Will match a known snapshot (0,0). 1`] = `0`;

exports[`example/math.ts add3 Will match a known snapshot (0,1). 1`] = `1`;

exports[`example/math.ts add3 Will match a known snapshot (0,42). 1`] = `42`;

exports[`example/math.ts add3 Will match a known snapshot (0,42). 2`] = `42`;

exports[`example/math.ts add3 Will match a known snapshot (0,42). 3`] = `42`;

exports[`example/math.ts add3 Will match a known snapshot (0,NaN). 1`] = `NaN`;

exports[`example/math.ts add3 Will match a known snapshot (1,-1). 1`] = `0`;

exports[`example/math.ts add3 Will match a known snapshot (1,0). 1`] = `1`;

exports[`example/math.ts add3 Will match a known snapshot (1,1). 1`] = `2`;

exports[`example/math.ts add3 Will match a known snapshot (1,42). 1`] = `43`;

exports[`example/math.ts add3 Will match a known snapshot (1,42). 2`] = `43`;

exports[`example/math.ts add3 Will match a known snapshot (1,42). 3`] = `43`;

exports[`example/math.ts add3 Will match a known snapshot (1,NaN). 1`] = `NaN`;

exports[`example/math.ts add3 Will match a known snapshot (42,-1). 1`] = `41`;

exports[`example/math.ts add3 Will match a known snapshot (42,-1). 2`] = `41`;

exports[`example/math.ts add3 Will match a known snapshot (42,-1). 3`] = `41`;

exports[`example/math.ts add3 Will match a known snapshot (42,0). 1`] = `42`;

exports[`example/math.ts add3 Will match a known snapshot (42,0). 2`] = `42`;

exports[`example/math.ts add3 Will match a known snapshot (42,0). 3`] = `42`;

exports[`example/math.ts add3 Will match a known snapshot (42,1). 1`] = `43`;

exports[`example/math.ts add3 Will match a known snapshot (42,1). 2`] = `43`;

exports[`example/math.ts add3 Will match a known snapshot (42,1). 3`] = `43`;

exports[`example/math.ts add3 Will match a known snapshot (42,42). 1`] = `84`;

exports[`example/math.ts add3 Will match a known snapshot (42,42). 2`] = `84`;

exports[`example/math.ts add3 Will match a known snapshot (42,42). 3`] = `84`;

exports[`example/math.ts add3 Will match a known snapshot (42,42). 4`] = `84`;

exports[`example/math.ts add3 Will match a known snapshot (42,42). 5`] = `84`;

exports[`example/math.ts add3 Will match a known snapshot (42,42). 6`] = `84`;

exports[`example/math.ts add3 Will match a known snapshot (42,42). 7`] = `84`;

exports[`example/math.ts add3 Will match a known snapshot (42,42). 8`] = `84`;

exports[`example/math.ts add3 Will match a known snapshot (42,42). 9`] = `84`;

exports[`example/math.ts add3 Will match a known snapshot (42,NaN). 1`] = `NaN`;

exports[`example/math.ts add3 Will match a known snapshot (42,NaN). 2`] = `NaN`;

exports[`example/math.ts add3 Will match a known snapshot (42,NaN). 3`] = `NaN`;

exports[`example/math.ts add3 Will match a known snapshot (NaN,-1). 1`] = `NaN`;

exports[`example/math.ts add3 Will match a known snapshot (NaN,0). 1`] = `NaN`;

exports[`example/math.ts add3 Will match a known snapshot (NaN,1). 1`] = `NaN`;

exports[`example/math.ts add3 Will match a known snapshot (NaN,42). 1`] = `NaN`;

exports[`example/math.ts add3 Will match a known snapshot (NaN,42). 2`] = `NaN`;

exports[`example/math.ts add3 Will match a known snapshot (NaN,42). 3`] = `NaN`;

exports[`example/math.ts add3 Will match a known snapshot (NaN,NaN). 1`] = `NaN`;

exports[`example/math.ts pointDistance Will match a known snapshot ({ x: 1, y: 2 },{ x: 1, y: 2 }). 1`] = `0`;

exports[`example/math.ts pointDistance Will match a known snapshot ({ x: 1, y: 2 },{ x: 3, y: 4 }). 1`] = `2`;

exports[`example/math.ts pointDistance Will match a known snapshot ({ x: 3, y: 4 },{ x: 1, y: 2 }). 1`] = `2`;

exports[`example/math.ts pointDistance Will match a known snapshot ({ x: 3, y: 4 },{ x: 3, y: 4 }). 1`] = `0`;

exports[`example/words.ts add Will match a known snapshot ("Matthew Carter","Matthew Carter"). 1`] = `"Matthew CarterMatthew Carter"`;

exports[`example/words.ts add Will match a known snapshot ("Matthew Carter","a,b,c,d"). 1`] = `"Matthew Cartera,b,c,d"`;

exports[`example/words.ts add Will match a known snapshot ("Matthew Carter","hello world"). 1`] = `"Matthew Carterhello world"`;

exports[`example/words.ts add Will match a known snapshot ("Matthew Carter","tHiS iS sOmE cRaZy fOrMaT"). 1`] = `"Matthew CartertHiS iS sOmE cRaZy fOrMaT"`;

exports[`example/words.ts add Will match a known snapshot ("a,b,c,d","Matthew Carter"). 1`] = `"a,b,c,dMatthew Carter"`;

exports[`example/words.ts add Will match a known snapshot ("a,b,c,d","a,b,c,d"). 1`] = `"a,b,c,da,b,c,d"`;

exports[`example/words.ts add Will match a known snapshot ("a,b,c,d","hello world"). 1`] = `"a,b,c,dhello world"`;

exports[`example/words.ts add Will match a known snapshot ("a,b,c,d","tHiS iS sOmE cRaZy fOrMaT"). 1`] = `"a,b,c,dtHiS iS sOmE cRaZy fOrMaT"`;

exports[`example/words.ts add Will match a known snapshot ("hello world","Matthew Carter"). 1`] = `"hello worldMatthew Carter"`;

exports[`example/words.ts add Will match a known snapshot ("hello world","a,b,c,d"). 1`] = `"hello worlda,b,c,d"`;

exports[`example/words.ts add Will match a known snapshot ("hello world","hello world"). 1`] = `"hello worldhello world"`;

exports[`example/words.ts add Will match a known snapshot ("hello world","tHiS iS sOmE cRaZy fOrMaT"). 1`] = `"hello worldtHiS iS sOmE cRaZy fOrMaT"`;

exports[`example/words.ts add Will match a known snapshot ("tHiS iS sOmE cRaZy fOrMaT","Matthew Carter"). 1`] = `"tHiS iS sOmE cRaZy fOrMaTMatthew Carter"`;

exports[`example/words.ts add Will match a known snapshot ("tHiS iS sOmE cRaZy fOrMaT","a,b,c,d"). 1`] = `"tHiS iS sOmE cRaZy fOrMaTa,b,c,d"`;

exports[`example/words.ts add Will match a known snapshot ("tHiS iS sOmE cRaZy fOrMaT","hello world"). 1`] = `"tHiS iS sOmE cRaZy fOrMaThello world"`;

exports[`example/words.ts add Will match a known snapshot ("tHiS iS sOmE cRaZy fOrMaT","tHiS iS sOmE cRaZy fOrMaT"). 1`] = `"tHiS iS sOmE cRaZy fOrMaTtHiS iS sOmE cRaZy fOrMaT"`;

exports[`example/words.ts add2 Will match a known snapshot ("Matthew Carter","Matthew Carter"). 1`] = `"Matthew CarterMatthew Carter"`;

exports[`example/words.ts add2 Will match a known snapshot ("Matthew Carter","a,b,c,d"). 1`] = `"Matthew Cartera,b,c,d"`;

exports[`example/words.ts add2 Will match a known snapshot ("Matthew Carter","hello world"). 1`] = `"Matthew Carterhello world"`;

exports[`example/words.ts add2 Will match a known snapshot ("Matthew Carter","tHiS iS sOmE cRaZy fOrMaT"). 1`] = `"Matthew CartertHiS iS sOmE cRaZy fOrMaT"`;

exports[`example/words.ts add2 Will match a known snapshot ("a,b,c,d","Matthew Carter"). 1`] = `"a,b,c,dMatthew Carter"`;

exports[`example/words.ts add2 Will match a known snapshot ("a,b,c,d","a,b,c,d"). 1`] = `"a,b,c,da,b,c,d"`;

exports[`example/words.ts add2 Will match a known snapshot ("a,b,c,d","hello world"). 1`] = `"a,b,c,dhello world"`;

exports[`example/words.ts add2 Will match a known snapshot ("a,b,c,d","tHiS iS sOmE cRaZy fOrMaT"). 1`] = `"a,b,c,dtHiS iS sOmE cRaZy fOrMaT"`;

exports[`example/words.ts add2 Will match a known snapshot ("hello world","Matthew Carter"). 1`] = `"hello worldMatthew Carter"`;

exports[`example/words.ts add2 Will match a known snapshot ("hello world","a,b,c,d"). 1`] = `"hello worlda,b,c,d"`;

exports[`example/words.ts add2 Will match a known snapshot ("hello world","hello world"). 1`] = `"hello worldhello world"`;

exports[`example/words.ts add2 Will match a known snapshot ("hello world","tHiS iS sOmE cRaZy fOrMaT"). 1`] = `"hello worldtHiS iS sOmE cRaZy fOrMaT"`;

exports[`example/words.ts add2 Will match a known snapshot ("tHiS iS sOmE cRaZy fOrMaT","Matthew Carter"). 1`] = `"tHiS iS sOmE cRaZy fOrMaTMatthew Carter"`;

exports[`example/words.ts add2 Will match a known snapshot ("tHiS iS sOmE cRaZy fOrMaT","a,b,c,d"). 1`] = `"tHiS iS sOmE cRaZy fOrMaTa,b,c,d"`;

exports[`example/words.ts add2 Will match a known snapshot ("tHiS iS sOmE cRaZy fOrMaT","hello world"). 1`] = `"tHiS iS sOmE cRaZy fOrMaThello world"`;

exports[`example/words.ts add2 Will match a known snapshot ("tHiS iS sOmE cRaZy fOrMaT","tHiS iS sOmE cRaZy fOrMaT"). 1`] = `"tHiS iS sOmE cRaZy fOrMaTtHiS iS sOmE cRaZy fOrMaT"`;

exports[`example/words.ts add3 Will match a known snapshot ("Matthew Carter","Matthew Carter"). 1`] = `"Matthew CarterMatthew Carter"`;

exports[`example/words.ts add3 Will match a known snapshot ("Matthew Carter","a,b,c,d"). 1`] = `"Matthew Cartera,b,c,d"`;

exports[`example/words.ts add3 Will match a known snapshot ("Matthew Carter","hello world"). 1`] = `"Matthew Carterhello world"`;

exports[`example/words.ts add3 Will match a known snapshot ("Matthew Carter","tHiS iS sOmE cRaZy fOrMaT"). 1`] = `"Matthew CartertHiS iS sOmE cRaZy fOrMaT"`;

exports[`example/words.ts add3 Will match a known snapshot ("a,b,c,d","Matthew Carter"). 1`] = `"a,b,c,dMatthew Carter"`;

exports[`example/words.ts add3 Will match a known snapshot ("a,b,c,d","a,b,c,d"). 1`] = `"a,b,c,da,b,c,d"`;

exports[`example/words.ts add3 Will match a known snapshot ("a,b,c,d","hello world"). 1`] = `"a,b,c,dhello world"`;

exports[`example/words.ts add3 Will match a known snapshot ("a,b,c,d","tHiS iS sOmE cRaZy fOrMaT"). 1`] = `"a,b,c,dtHiS iS sOmE cRaZy fOrMaT"`;

exports[`example/words.ts add3 Will match a known snapshot ("hello world","Matthew Carter"). 1`] = `"hello worldMatthew Carter"`;

exports[`example/words.ts add3 Will match a known snapshot ("hello world","a,b,c,d"). 1`] = `"hello worlda,b,c,d"`;

exports[`example/words.ts add3 Will match a known snapshot ("hello world","hello world"). 1`] = `"hello worldhello world"`;

exports[`example/words.ts add3 Will match a known snapshot ("hello world","tHiS iS sOmE cRaZy fOrMaT"). 1`] = `"hello worldtHiS iS sOmE cRaZy fOrMaT"`;

exports[`example/words.ts add3 Will match a known snapshot ("tHiS iS sOmE cRaZy fOrMaT","Matthew Carter"). 1`] = `"tHiS iS sOmE cRaZy fOrMaTMatthew Carter"`;

exports[`example/words.ts add3 Will match a known snapshot ("tHiS iS sOmE cRaZy fOrMaT","a,b,c,d"). 1`] = `"tHiS iS sOmE cRaZy fOrMaTa,b,c,d"`;

exports[`example/words.ts add3 Will match a known snapshot ("tHiS iS sOmE cRaZy fOrMaT","hello world"). 1`] = `"tHiS iS sOmE cRaZy fOrMaThello world"`;

exports[`example/words.ts add3 Will match a known snapshot ("tHiS iS sOmE cRaZy fOrMaT","tHiS iS sOmE cRaZy fOrMaT"). 1`] = `"tHiS iS sOmE cRaZy fOrMaTtHiS iS sOmE cRaZy fOrMaT"`;

exports[`example/words.ts add4 Will match a known snapshot (-1,"Matthew Carter"). 1`] = `"-1Matthew Carter"`;

exports[`example/words.ts add4 Will match a known snapshot (-1,"a,b,c,d"). 1`] = `"-1a,b,c,d"`;

exports[`example/words.ts add4 Will match a known snapshot (-1,"hello world"). 1`] = `"-1hello world"`;

exports[`example/words.ts add4 Will match a known snapshot (-1,"tHiS iS sOmE cRaZy fOrMaT"). 1`] = `"-1tHiS iS sOmE cRaZy fOrMaT"`;

exports[`example/words.ts add4 Will match a known snapshot (0,"Matthew Carter"). 1`] = `"0Matthew Carter"`;

exports[`example/words.ts add4 Will match a known snapshot (0,"a,b,c,d"). 1`] = `"0a,b,c,d"`;

exports[`example/words.ts add4 Will match a known snapshot (0,"hello world"). 1`] = `"0hello world"`;

exports[`example/words.ts add4 Will match a known snapshot (0,"tHiS iS sOmE cRaZy fOrMaT"). 1`] = `"0tHiS iS sOmE cRaZy fOrMaT"`;

exports[`example/words.ts add4 Will match a known snapshot (1,"Matthew Carter"). 1`] = `"1Matthew Carter"`;

exports[`example/words.ts add4 Will match a known snapshot (1,"a,b,c,d"). 1`] = `"1a,b,c,d"`;

exports[`example/words.ts add4 Will match a known snapshot (1,"hello world"). 1`] = `"1hello world"`;

exports[`example/words.ts add4 Will match a known snapshot (1,"tHiS iS sOmE cRaZy fOrMaT"). 1`] = `"1tHiS iS sOmE cRaZy fOrMaT"`;

exports[`example/words.ts add4 Will match a known snapshot (42,"Matthew Carter"). 1`] = `"42Matthew Carter"`;

exports[`example/words.ts add4 Will match a known snapshot (42,"Matthew Carter"). 2`] = `"42Matthew Carter"`;

exports[`example/words.ts add4 Will match a known snapshot (42,"Matthew Carter"). 3`] = `"42Matthew Carter"`;

exports[`example/words.ts add4 Will match a known snapshot (42,"a,b,c,d"). 1`] = `"42a,b,c,d"`;

exports[`example/words.ts add4 Will match a known snapshot (42,"a,b,c,d"). 2`] = `"42a,b,c,d"`;

exports[`example/words.ts add4 Will match a known snapshot (42,"a,b,c,d"). 3`] = `"42a,b,c,d"`;

exports[`example/words.ts add4 Will match a known snapshot (42,"hello world"). 1`] = `"42hello world"`;

exports[`example/words.ts add4 Will match a known snapshot (42,"hello world"). 2`] = `"42hello world"`;

exports[`example/words.ts add4 Will match a known snapshot (42,"hello world"). 3`] = `"42hello world"`;

exports[`example/words.ts add4 Will match a known snapshot (42,"tHiS iS sOmE cRaZy fOrMaT"). 1`] = `"42tHiS iS sOmE cRaZy fOrMaT"`;

exports[`example/words.ts add4 Will match a known snapshot (42,"tHiS iS sOmE cRaZy fOrMaT"). 2`] = `"42tHiS iS sOmE cRaZy fOrMaT"`;

exports[`example/words.ts add4 Will match a known snapshot (42,"tHiS iS sOmE cRaZy fOrMaT"). 3`] = `"42tHiS iS sOmE cRaZy fOrMaT"`;

exports[`example/words.ts add4 Will match a known snapshot (NaN,"Matthew Carter"). 1`] = `"NaNMatthew Carter"`;

exports[`example/words.ts add4 Will match a known snapshot (NaN,"a,b,c,d"). 1`] = `"NaNa,b,c,d"`;

exports[`example/words.ts add4 Will match a known snapshot (NaN,"hello world"). 1`] = `"NaNhello world"`;

exports[`example/words.ts add4 Will match a known snapshot (NaN,"tHiS iS sOmE cRaZy fOrMaT"). 1`] = `"NaNtHiS iS sOmE cRaZy fOrMaT"`;

exports[`example/words.ts adder Will match a known snapshot ("Matthew Carter","Matthew Carter"). 1`] = `"Matthew CarterMatthew Carter"`;

exports[`example/words.ts adder Will match a known snapshot ("Matthew Carter","a,b,c,d"). 1`] = `"Matthew Cartera,b,c,d"`;

exports[`example/words.ts adder Will match a known snapshot ("Matthew Carter","hello world"). 1`] = `"Matthew Carterhello world"`;

exports[`example/words.ts adder Will match a known snapshot ("Matthew Carter","tHiS iS sOmE cRaZy fOrMaT"). 1`] = `"Matthew CartertHiS iS sOmE cRaZy fOrMaT"`;

exports[`example/words.ts adder Will match a known snapshot ("a,b,c,d","Matthew Carter"). 1`] = `"a,b,c,dMatthew Carter"`;

exports[`example/words.ts adder Will match a known snapshot ("a,b,c,d","a,b,c,d"). 1`] = `"a,b,c,da,b,c,d"`;

exports[`example/words.ts adder Will match a known snapshot ("a,b,c,d","hello world"). 1`] = `"a,b,c,dhello world"`;

exports[`example/words.ts adder Will match a known snapshot ("a,b,c,d","tHiS iS sOmE cRaZy fOrMaT"). 1`] = `"a,b,c,dtHiS iS sOmE cRaZy fOrMaT"`;

exports[`example/words.ts adder Will match a known snapshot ("hello world","Matthew Carter"). 1`] = `"hello worldMatthew Carter"`;

exports[`example/words.ts adder Will match a known snapshot ("hello world","a,b,c,d"). 1`] = `"hello worlda,b,c,d"`;

exports[`example/words.ts adder Will match a known snapshot ("hello world","hello world"). 1`] = `"hello worldhello world"`;

exports[`example/words.ts adder Will match a known snapshot ("hello world","tHiS iS sOmE cRaZy fOrMaT"). 1`] = `"hello worldtHiS iS sOmE cRaZy fOrMaT"`;

exports[`example/words.ts adder Will match a known snapshot ("tHiS iS sOmE cRaZy fOrMaT","Matthew Carter"). 1`] = `"tHiS iS sOmE cRaZy fOrMaTMatthew Carter"`;

exports[`example/words.ts adder Will match a known snapshot ("tHiS iS sOmE cRaZy fOrMaT","a,b,c,d"). 1`] = `"tHiS iS sOmE cRaZy fOrMaTa,b,c,d"`;

exports[`example/words.ts adder Will match a known snapshot ("tHiS iS sOmE cRaZy fOrMaT","hello world"). 1`] = `"tHiS iS sOmE cRaZy fOrMaThello world"`;

exports[`example/words.ts adder Will match a known snapshot ("tHiS iS sOmE cRaZy fOrMaT","tHiS iS sOmE cRaZy fOrMaT"). 1`] = `"tHiS iS sOmE cRaZy fOrMaTtHiS iS sOmE cRaZy fOrMaT"`;
